debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlDefaultValuePreConverter.java

```diff
@@ -161,7 +161,7 @@ private Object convertToDuration(Column column, String value) {
      * @return the converted value;
      */
     private Object convertToInteger(String value) {
-        return Integer.parseInt(value);
+        return Integer.valueOf(value);
```

Comment:
What's the reasoning behind this change?

Reply:
As you know `Integer.valueOf()`/`Short.valueOf()`/`Long.valueOf()` methods use cache and may return the cached value. I know Java compiler uses `*.valueOf()` methods for autoboxing, but if we already call `Integer.parseInt()`/`Short.parseShort()`/`Long.parseLong()` methods, then it is better to use `*.valueOf()` methods directly. Explicit is better than implicit.

Reply:
Ok.